elixir/sum-of-multiples/lib/sum_of_multiples.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SumOfMultiples do
+  @doc """
+  Adds up all numbers from 1 to a given end number that are multiples of the factors provided.
+  """
+
+
+  def to(limit, factors) do
+    Enum.reduce(0..limit-1, 0, & &2 + get_factor_sum(&1, factors))
+
+
+  end
+
+  def get_factor_sum(number, factors) do
+    Enum.map(factors, fn(x) ->
+      cond do
+        rem(number, x) == 0 -> number
+        true -> 0
+      end
+    end)
```

Comment:
```suggestion
    factors
    |> Enum.filter(& rem(number, &1) == 0)
```

Try to always start your pipeline with a value rather than a function where possible.

Also that map looks like it should be a filter!

Reply:
I started the pipeline out right!
BUT: I cannot use filter here, because I am using the map on my list of factors, but I am not keeping and summing up factors, but the numbers. If I use filter, I'd keep the factor, not the number.